### ISSUE TITLE
UI Redesign - Resizability

### ISSUE THREAD

**Issue by collectivenectar** [collectivenectar/pengen#11]
Currently the UI lacks flexibility/accessibility, enough to limit functionality when the screen width gets small enough. Add some elements and css to increase UI flexibility so that a user can resize the window and not break the app. It would be great to have the toolbar be more flexible as well.

**Comment by collectivenectar** [comment]
Added two queries for landscape/portrait orientation(configured by h/w ratio not by device orientation)
and set some basic css for each to make it look a little better. I also added a function so that the canvas object size is set according to a standard % of window.innerWidth and window.innerHeight so it should do a little better. If this is good enough for you @moogwol it's good enough for me, this is my first time fiddling with queries so I don't know if this issue can be said to be closed.

**Comment by moogwol** [comment]
I just tried out the changes on the  ipad emulator and they work and look good so I think we can call this issue closed
